streams-tests/shared/src/test/scala/zio/stream/ZTransducerSpec.scala

```diff
@@ -67,17 +67,19 @@ object ZTransducerSpec extends ZIOBaseSpec {
     suite("Constructors")(
       suite("collectAllN")(
         testM("happy path") {
-          assertM(run(ZTransducer.collectAllN[Int](3), List(Chunk(1, 2, 3, 4))))(equalTo(Chunk(List(1, 2, 3), List(4))))
+          assertM(run(ZTransducer.collectAllN[Int](3), List(Chunk(1, 2, 3, 4))))(
```

Comment:
@svranesevic Last request and then this is good to merge: the implementation is a bit more involved now, so can we add a property check? Just equivalence with `List#grouped` should be sufficient.